src/checkbox/src/Checkbox.js

```diff
@@ -136,10 +45,16 @@ class Checkbox extends PureComponent {
       onChange,
       value,
       indeterminate,
-      innerRef,
-      ...props
-    } = this.props
+      ...rest
+    } = props
+
+    useEffect(() => {
+      if (ref?.current) {
```

Comment:
Oh shoot... this won't compile? I was stuck in TypeScript mode. We will have to use `ref && ref.current` instead.

Reply:
Ah yeah. Might be worth upgrading Babel to support optional chaining?